torch/onnx/symbolic_opset9.py

```diff
@@ -195,6 +195,11 @@ def sign(g, self):
 
 
 def _slice_op(g, input, axes, starts, ends):
+    if sym_help._export_onnx_opset_version != 9:
```

Comment:
this is very misleading... can you add some comments talking about why we need other version of `_slice_op`?